pkg/model/estimator/local/benchmark_test.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package local_test
+
+import (
+	"strconv"
+	"testing"
+
+	collector_metric "github.com/sustainable-computing-io/kepler/pkg/collector/metric"
+	"github.com/sustainable-computing-io/kepler/pkg/config"
+	"github.com/sustainable-computing-io/kepler/pkg/model/estimator/local"
+	"github.com/sustainable-computing-io/kepler/pkg/power/components/source"
+)
+
+func benchmarkNtesting(b *testing.B, continerNumber int) {
+	containersMetrics := map[string]*collector_metric.ContainerMetrics{}
+	nodeMetrics := collector_metric.NewNodeMetrics()
+	collector_metric.ContainerMetricNames = []string{config.CoreUsageMetric}
+	nodeMetrics.AddNodeResUsageFromContainerResUsage(containersMetrics)
+
+	nodePlatformEnergy := map[string]float64{}
+	// initialize the node energy with aggregated energy, which will be used to calculate delta energy
+	nodePlatformEnergy["sensor0"] = 10
+	nodeMetrics.SetLastestPlatformEnergy(nodePlatformEnergy)
+	nodeMetrics.UpdateIdleEnergy()
+	// the second node energy will represent the idle and dynamic power
+	nodePlatformEnergy["sensor0"] = 20 // 5J idle, 5J dynamic power
```

Comment:
didn't get = 20 then // 5J idle, 5J dynamic, what's the relationship between them? how it's calculated?

Reply:
well, in fact those code parts is reuse from the unit test. and I didn't clean up the comments.
if the unit test fail the test case will broken, hence didn't re check the value in benchmark testing.
I will adjust those part in coming patch, but don't have plan to refactor test code as common function for now.

Reply:
ok, not a big deal , though hard for me to understand the logic behind it :) 
so can you help check above comments then we can merge it?

Reply:
comments removed.

Reply:
@jichenjc , you are able to check the test logic at file ratio_test.go

Reply:
got it , thanks